source/common/ratelimit/ratelimit_impl.cc

```diff
@@ -15,10 +15,13 @@
 namespace Envoy {
 namespace RateLimit {
 
+static const std::string ratelimit_method_name =
```

Comment:
nit: dont think you need this as a separate variable, if you are using it only in one place.